.travis.yml

```diff
@@ -8,10 +8,22 @@ install:
   - go get -u github.com/golang/lint/golint
   - go get -u github.com/Masterminds/glide
   - go get -u golang.org/x/net/context
+  - go get -u github.com/HewlettPackard/gas
   - export GO15VENDOREXPERIMENT=1
   - glide install
 
 script:
+  # GAS commands for SDL scanning
+  - (cd arm && gas -fmt=json -out=results.json ./... > /dev/null)
```

Comment:
> The reason this is broken out per-folder is because the build process pulls in 3rd party code under the 'vendor' folder, and this code has additional findings that we're not triaging at this time.

we can easily fix this by using `gas $(go list ./... | grep -v 'vendor')`.